Regression test run failed when compacting along with drop collection caused by server crashed

**Please state your issue using the following template and, most importantly, in English.**

**Describe the bug**

```
[gw2] [  0%] PASSED test_compact.py::TestCompactBase::test_insert_delete_part_and_compact_threshold 
test_compact.py::TestCompactBase::test_compact_after_index_created[get_simple_index1] 
[gw1] [  0%] PASSED test_compact.py::TestCompactBase::test_insert_delete_part_and_compact 
test_compact.py::TestCompactBase::test_compact_after_index_created[get_simple_index0] 
[gw3] [  0%] PASSED test_compact.py::TestCompactBase::test_insert_delete_all_and_compact 
[gw3] [  0%] ERROR test_compact.py::TestCompactBase::test_insert_delete_all_and_compact 
test_compact.py::TestCompactBase::test_compact_after_index_created[get_simple_index2] 
[gw3] node down: keyboard-interrupt
[gw3] [  0%] FAILED test_compact.py::TestCompactBase::test_compact_after_index_created[get_simple_index2] 
Replacing crashed worker gw3

[gw4] linux Python 3.6.8 cwd: /home/jenkins/agent/workspace/milvus-ci_0.11.1/tests/milvus_python_test

==================================== ERRORS ====================================
___ ERROR at teardown of TestCompactBase.test_insert_delete_all_and_compact ____
[gw3] linux -- Python 3.6.8 /usr/local/bin/python3

tp = <class 'grpc._channel._MultiThreadedRendezvous'>, value = None, tb = None

    def reraise(tp, value, tb=None):
        try:
            if value is None:
                value = tp()
            if value.__traceback__ is not tb:
                raise value.with_traceback(tb)
>           raise value

/usr/local/lib/python3.6/site-packages/six.py:703: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/usr/local/lib/python3.6/site-packages/six.py:703: in reraise
    raise value
/usr/local/lib/python3.6/site-packages/six.py:703: in reraise
    raise value
/usr/local/lib/python3.6/site-packages/allure_commons/_allure.py:181: in __call__
    return self._fixture_function(*args, **kwargs)
conftest.py:137: in teardown
    connect.drop_collection(collection_name, timeout=delete_timeout)
/usr/local/lib/python3.6/site-packages/milvus/client/stub.py:35: in inner
    return func(self, *args, **kwargs)
/usr/local/lib/python3.6/site-packages/milvus/client/stub.py:328: in drop_collection
    return handler.drop_collection(collection_name, timeout)
/usr/local/lib/python3.6/site-packages/milvus/client/grpc_handler.py:63: in handler
    raise e
/usr/local/lib/python3.6/site-packages/milvus/client/grpc_handler.py:44: in handler
    return func(self, *args, **kwargs)
/usr/local/lib/python3.6/site-packages/milvus/client/grpc_handler.py:336: in drop_collection
    status = rf.result()
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <_MultiThreadedRendezvous of RPC that terminated with:
	status = StatusCode.DEADLINE_EXCEEDED
	details = "Deadline Exc...99:19530","file":"src/core/lib/surface/call.cc","file_line":1062,"grpc_message":"Deadline Exceeded","grpc_status":4}"
>
timeout = None

    def result(self, timeout=None):
        """Returns the result of the computation or raises its exception.
    
        See grpc.Future.result for the full API contract.
        """
        with self._state.condition:
            timed_out = _common.wait(self._state.condition.wait,
                                     self._is_complete,
                                     timeout=timeout)
            if timed_out:
                raise grpc.FutureTimeoutError()
            else:
                if self._state.code is grpc.StatusCode.OK:
                    return self._state.response
                elif self._state.cancelled:
                    raise grpc.FutureCancelledError()
                else:
>                   raise self
E                   grpc._channel._MultiThreadedRendezvous: <_MultiThreadedRendezvous of RPC that terminated with:
E                   	status = StatusCode.DEADLINE_EXCEEDED
E                   	details = "Deadline Exceeded"
E                   	debug_error_string = "{"created":"@1606151462.669800359","description":"Error received from peer ipv4:10.103.188.99:19530","file":"src/core/lib/surface/call.cc","file_line":1062,"grpc_message":"Deadline Exceeded","grpc_status":4}"
E                   >

/usr/local/lib/python3.6/site-packages/grpc/_channel.py:625: _MultiThreadedRendezvous
------------------------------ Captured log call -------------------------------
[2020-11-23 17:10:02-INFO-root]: [{'data_size': 0, 'id': 16, 'row_count': 0, 'segment_count': 0, 'segments': None, 'tag': '_default'}] (test_compact.py:226)
--------------------------- Captured stderr teardown ---------------------------

Addr [milvus-ci-0-11-1-74-single-centos7-cpu.milvus.svc.cluster.local:19530] drop_collection
RPC error: <_MultiThreadedRendezvous of RPC that terminated with:
	status = StatusCode.DEADLINE_EXCEEDED
	details = "Deadline Exceeded"
	debug_error_string = "{"created":"@1606151462.669800359","description":"Error received from peer ipv4:10.103.188.99:19530","file":"src/core/lib/surface/call.cc","file_line":1062,"grpc_message":"Deadline Exceeded","grpc_status":4}"
>
	{'API start': '2020-11-23 17:10:02.668229', 'RPC start': '2020-11-23 17:10:02.668852', 'RPC error': '2020-11-23 17:11:02.670658'}
---------------------------- Captured log teardown -----------------------------
[2020-11-23 17:11:02-ERROR-milvus.client.grpc_handler]: 
Addr [milvus-ci-0-11-1-74-single-centos7-cpu.milvus.svc.cluster.local:19530] drop_collection
RPC error: <_MultiThreadedRendezvous of RPC that terminated with:
	status = StatusCode.DEADLINE_EXCEEDED
	details = "Deadline Exceeded"
	debug_error_string = "{"created":"@1606151462.669800359","description":"Error received from peer ipv4:10.103.188.99:19530","file":"src/core/lib/surface/call.cc","file_line":1062,"grpc_message":"Deadline Exceeded","grpc_status":4}"
>
	{'API start': '2020-11-23 17:10:02.668229', 'RPC start': '2020-11-23 17:10:02.668852', 'RPC error': '2020-11-23 17:11:02.670658'} (grpc_handler.py:61)
```
```
[2020-11-24 18:43:54,879][INFO][SERVER][Flush][grpcpp_sync_ser] Request [106] Flush end.
[2020-11-24 18:43:54,887][INFO][SERVER][Flush][grpcpp_sync_ser] Request [107] Flush begin.
[2020-11-24 18:43:54,888][INFO][SERVER][CreateCollection][grpcpp_sync_ser] Request [108] CreateCollection begin.
[2020-11-24 18:43:54,888][INFO][SERVER][Flush][grpcpp_sync_ser] Request [109] Flush begin.
[2020-11-24 18:43:54,888][INFO][SERVER][ShowCollectionInfo][grpcpp_sync_ser] Request [110] ShowCollectionInfo begin.
[2020-11-24 18:43:54,888][INFO][SERVER][ShowCollectionInfo][grpcpp_sync_ser] Request [110] ShowCollectionInfo end.
[2020-11-24 18:43:54,889][INFO][SERVER][DeleteByID][grpcpp_sync_ser] Request [111] DeleteByID begin.
[2020-11-24 18:43:54,890][INFO][SERVER][Flush][grpcpp_sync_ser] Request [107] Flush end.
[2020-11-24 18:43:54,890][INFO][SERVER][Compact][grpcpp_sync_ser] Request [112] Compact begin.
[2020-11-24 18:43:54,892][INFO][SERVER][CreateCollection][grpcpp_sync_ser] Request [108] CreateCollection end.
[2020-11-24 18:43:54,893][INFO][SERVER][HasCollection][grpcpp_sync_ser] Request [113] HasCollection begin.
[2020-11-24 18:43:54,893][INFO][SERVER][HasCollection][grpcpp_sync_ser] Request [113] HasCollection end.
[2020-11-24 18:43:54,901][INFO][SERVER][Flush][grpcpp_sync_ser] Request [109] Flush end.
[2020-11-24 18:43:54,901][INFO][SERVER][DeleteByID][grpcpp_sync_ser] Request [111] DeleteByID end.
[2020-11-24 18:43:54,902][INFO][SERVER][ShowCollectionInfo][grpcpp_sync_ser] Request [114] ShowCollectionInfo begin.
[2020-11-24 18:43:54,902][INFO][SERVER][ShowCollectionInfo][grpcpp_sync_ser] Request [114] ShowCollectionInfo end.
[2020-11-24 18:43:54,902][INFO][SERVER][Flush][grpcpp_sync_ser] Request [115] Flush begin.
[2020-11-24 18:43:54,903][INFO][SERVER][DeleteByID][grpcpp_sync_ser] Request [116] DeleteByID begin.
[2020-11-24 18:43:54,905][INFO][SERVER][HandleSignal][create_index] Server received critical signal: 11
[2020-11-24 18:43:54,909][INFO][SERVER][PrintStacktrace][create_index] Call stack:
[2020-11-24 18:43:54,909][INFO][SERVER][PrintStacktrace][create_index] /var/lib/milvus/bin/milvus_server(_ZN6milvus15PrintStacktraceEv+0x26) [0x111e866]
[2020-11-24 18:43:54,909][INFO][SERVER][PrintStacktrace][create_index] /var/lib/milvus/bin/milvus_server(_ZN6milvus12HandleSignalEi+0x190) [0x111f3d0]
[2020-11-24 18:43:54,909][INFO][SERVER][PrintStacktrace][create_index] /lib64/libc.so.6(+0x36400) [0x7f3b28b8c400]
[2020-11-24 18:43:54,909][INFO][SERVER][PrintStacktrace][create_index] /lib64/libc.so.6(abort+0x297) [0x7f3b28b8dbc7]
[2020-11-24 18:43:54,909][INFO][SERVER][PrintStacktrace][create_index] /lib64/libc.so.6(+0x78ed7) [0x7f3b28bceed7]
[2020-11-24 18:43:54,909][INFO][SERVER][PrintStacktrace][create_index] /lib64/libc.so.6(+0x7f3e4) [0x7f3b28bd53e4]
[2020-11-24 18:43:54,909][INFO][SERVER][PrintStacktrace][create_index] /lib64/libc.so.6(+0x82b20) [0x7f3b28bd8b20]
[2020-11-24 18:43:54,909][INFO][SERVER][PrintStacktrace][create_index] /lib64/libc.so.6(__libc_malloc+0x4c) [0x7f3b28bdb6fc]
[2020-11-24 18:43:54,909][INFO][SERVER][PrintStacktrace][create_index] /lib64/libstdc++.so.6(_Znwm+0x1d) [0x7f3b2949aecd]
[2020-11-24 18:43:54,909][INFO][SERVER][PrintStacktrace][create_index] /var/lib/milvus/bin/milvus_server(_ZN6milvus6engine8snapshot16OperationContextC2ERKS2_+0x4c2) [0x93bd62]
[2020-11-24 18:43:54,909][INFO][SERVER][PrintStacktrace][create_index] /var/lib/milvus/bin/milvus_server(_ZN6milvus6engine8snapshot10OperationsC2ERKNS1_16OperationContextENS1_14ScopedResourceINS1_8SnapshotEEERKNS1_14OperationsTypeE+0x41) [0x996421]
[2020-11-24 18:43:54,909][INFO][SERVER][PrintStacktrace][create_index] /var/lib/milvus/bin/milvus_server(_ZN6milvus6engine8snapshot14MergeOperationC2ERKNS1_16OperationContextENS1_14ScopedResourceINS1_8SnapshotEEE+0x4b) [0x95166b]
[2020-11-24 18:43:54,909][INFO][SERVER][PrintStacktrace][create_index] /var/lib/milvus/bin/milvus_server(_ZN6milvus6engine9MergeTask7ExecuteEv+0x54c) [0x94b29c]
[2020-11-24 18:43:54,909][INFO][SERVER][PrintStacktrace][create_index] /var/lib/milvus/bin/milvus_server(_ZN6milvus6engine6DBImpl7CompactERKSt10shared_ptrINS_6server7ContextEERKSsd+0x8da) [0x8f2aba]
[2020-11-24 18:43:54,909][INFO][SERVER][PrintStacktrace][create_index] /var/lib/milvus/bin/milvus_server(_ZN6milvus6engine7DBProxy7CompactERKSt10shared_ptrINS_6server7ContextEERKSsd+0xb3) [0xa60a03]
[2020-11-24 18:43:54,909][INFO][SERVER][PrintStacktrace][create_index] /var/lib/milvus/bin/milvus_server(_ZN6milvus6server10CompactReq9OnExecuteEv+0x30c) [0x8cd76c]
[2020-11-24 18:43:54,909][INFO][SERVER][PrintStacktrace][create_index] /var/lib/milvus/bin/milvus_server(_ZN6milvus6server7BaseReq7ExecuteEv+0x20) [0x8cb010]
[2020-11-24 18:43:54,909][INFO][SERVER][PrintStacktrace][create_index] /var/lib/milvus/bin/milvus_server(_ZN6milvus6server12ReqScheduler13TakeToExecuteESt10shared_ptrINS0_8ReqQueueEE+0x12d) [0x8c75cd]
[2020-11-24 18:43:54,909][INFO][SERVER][PrintStacktrace][create_index] /var/lib/milvus/bin/milvus_server(_ZNSt6thread11_State_implINS_8_InvokerISt5tupleIJMN6milvus6server12ReqSchedulerEFvSt10shared_ptrINS4_8ReqQueueEEEPS5_S8_EEEEE6_M_runEv+0x47) [0x8c9b77]
[2020-11-24 18:43:54,909][INFO][SERVER][PrintStacktrace][create_index] /var/lib/milvus/bin/milvus_server() [0x112c7ef]
[2020-11-24 18:43:54,909][INFO][SERVER][PrintStacktrace][create_index] /lib64/libpthread.so.0(+0x7ea5) [0x7f3b29bbbea5]
[2020-11-24 18:43:54,909][INFO][SERVER][PrintStacktrace][create_index] /lib64/libc.so.6(clone+0x6d) [0x7f3b28c548dd]
[2020-11-24 18:43:54,915][INFO][SERVER][Insert][grpcpp_sync_ser] Request [117] Insert begin.

```
**Steps/Code to reproduce behavior**
http://internal.zilliz.com:18080/jenkins/job/milvus-ci/job/0.11.1/74/artifact/_artifacts/centos7/cpu/milvus_cpu_mysql_dev_test.log
```
*** Error in `/var/lib/milvus/bin/milvus_server': malloc(): smallbin double linked list corrupted: 0x00007f3acc000b30 ***
======= Backtrace: =========
/lib64/libc.so.6(+0x7f3e4)[0x7f3b28bd53e4]
/lib64/libc.so.6(+0x82b20)[0x7f3b28bd8b20]
/lib64/libc.so.6(__libc_malloc+0x4c)[0x7f3b28bdb6fc]
/lib64/libstdc++.so.6(_Znwm+0x1d)[0x7f3b2949aecd]
/var/lib/milvus/bin/milvus_server(_ZN6milvus6engine8snapshot16OperationContextC2ERKS2_+0x4c2)[0x93bd62]
/var/lib/milvus/bin/milvus_server(_ZN6milvus6engine8snapshot10OperationsC2ERKNS1_16OperationContextENS1_14ScopedResourceINS1_8SnapshotEEERKNS1_14OperationsTypeE+0x41)[0x996421]
/var/lib/milvus/bin/milvus_server(_ZN6milvus6engine8snapshot14MergeOperationC2ERKNS1_16OperationContextENS1_14ScopedResourceINS1_8SnapshotEEE+0x4b)[0x95166b]
/var/lib/milvus/bin/milvus_server(_ZN6milvus6engine9MergeTask7ExecuteEv+0x54c)[0x94b29c]
/var/lib/milvus/bin/milvus_server(_ZN6milvus6engine6DBImpl7CompactERKSt10shared_ptrINS_6server7ContextEERKSsd+0x8da)[0x8f2aba]
/var/lib/milvus/bin/milvus_server(_ZN6milvus6engine7DBProxy7CompactERKSt10shared_ptrINS_6server7ContextEERKSsd+0xb3)[0xa60a03]
/var/lib/milvus/bin/milvus_server(_ZN6milvus6server10CompactReq9OnExecuteEv+0x30c)[0x8cd76c]
/var/lib/milvus/bin/milvus_server(_ZN6milvus6server7BaseReq7ExecuteEv+0x20)[0x8cb010]
/var/lib/milvus/bin/milvus_server(_ZN6milvus6server12ReqScheduler13TakeToExecuteESt10shared_ptrINS0_8ReqQueueEE+0x12d)[0x8c75cd]
/var/lib/milvus/bin/milvus_server(_ZNSt6thread11_State_implINS_8_InvokerISt5tupleIJMN6milvus6server12ReqSchedulerEFvSt10shared_ptrINS4_8ReqQueueEEEPS5_S8_EEEEE6_M_runEv+0x47)[0x8c9b77]
/var/lib/milvus/bin/milvus_server[0x112c7ef]
/lib64/libpthread.so.0(+0x7ea5)[0x7f3b29bbbea5]
/lib64/libc.so.6(clone+0x6d)[0x7f3b28c548dd]
======= Memory map: ========
00400000-013af000 r-xp 00000000 103:02 8927665                           /var/lib/milvus/bin/milvus_server
015af000-015e4000 r--p 00faf000 103:02 8927665                           /var/lib/milvus/bin/milvus_server
015e4000-015ec000 rw-p 00fe4000 103:02 8927665                           /var/lib/milvus/bin/milvus_server
015ec000-015fb000 rw-p 00000000 00:00 0 
01ed2000-01f9b000 rw-p 00000000 00:00 0                                  [heap]
7f3a64000000-7f3a64021000 rw-p 00000000 00:00 0 
7f3a64021000-7f3a68000000 ---p 00000000 00:00 0 
7f3a6c000000-7f3a6c021000 rw-p 00000000 00:00 0 
7f3a6c021000-7f3a70000000 ---p 00000000 00:00 0 
7f3a74000000-7f3a74021000 rw-p 00000000 00:00 0 
7f3a74021000-7f3a78000000 ---p 00000000 00:00 0 
7f3a7c000000-7f3a7c021000 rw-p 00000000 00:00 0 
7f3a7c021000-7f3a80000000 ---p 00000000 00:00 0 
7f3a84000000-7f3a84021000 rw-p 00000000 00:00 0 
7f3a84021000-7f3a88000000 ---p 00000000 00:00 0 
7f3a88000000-7f3a88021000 rw-p 00000000 00:00 0 
7f3a88021000-7f3a8c000000 ---p 00000000 00:00 0 
7f3a8c000000-7f3a8c093000 rw-p 00000000 00:00 0 
7f3a8c093000-7f3a90000000 ---p 00000000 00:00 0 
7f3a90000000-7f3a90021000 rw-p 00000000 00:00 0 
7f3a90021000-7f3a94000000 ---p 00000000 00:00 0 
7f3a98000000-7f3a98021000 rw-p 00000000 00:00 0 
7f3a98021000-7f3a9c000000 ---p 00000000 00:00 0 
7f3a9c000000-7f3a9c021000 rw-p 00000000 00:00 0 
7f3a9c021000-7f3aa0000000 ---p 00000000 00:00 0 
7f3aa4000000-7f3aa4021000 rw-p 00000000 00:00 0 
7f3aa4021000-7f3aa8000000 ---p 00000000 00:00 0 
7f3aac000000-7f3aac021000 rw-p 00000000 00:00 0 
7f3aac021000-7f3ab0000000 ---p 00000000 00:00 0 
7f3ab1ffc000-7f3ab1ffd000 ---p 00000000 00:00 0 
7f3ab1ffd000-7f3ab27fd000 rw-p 00000000 00:00 0 
7f3ab27fd000-7f3ab27fe000 ---p 00000000 00:00 0 
7f3ab27fe000-7f3ab2ffe000 rw-p 00000000 00:00 0 
7f3ab2ffe000-7f3ab2fff000 ---p 00000000 00:00 0 
7f3ab2fff000-7f3ab37ff000 rw-p 00000000 00:00 0 
7f3ab37ff000-7f3ab3800000 ---p 00000000 00:00 0 
7f3ab3800000-7f3ab4000000 rw-p 00000000 00:00 0 
7f3ab4000000-7f3ab4021000 rw-p 00000000 00:00 0 
7f3ab4021000-7f3ab8000000 ---p 00000000 00:00 0 
7f3ab87f9000-7f3ab87fa000 ---p 00000000 00:00 0 
7f3ab87fa000-7f3ab8ffa000 rw-p 00000000 00:00 0 
7f3ab8ffa000-7f3ab8ffb000 ---p 00000000 00:00 0 
7f3ab8ffb000-7f3ab97fb000 rw-p 00000000 00:00 0 
7f3ab97fb000-7f3ab97fc000 ---p 00000000 00:00 0 
7f3ab97fc000-7f3ab9ffc000 rw-p 00000000 00:00 0 
7f3ab9ffc000-7f3ab9ffd000 ---p 00000000 00:00 0 
7f3ab9ffd000-7f3aba7fd000 rw-p 00000000 00:00 0 
7f3aba7fd000-7f3aba7fe000 ---p 00000000 00:00 0 
7f3aba7fe000-7f3abaffe000 rw-p 00000000 00:00 0 
7f3abaffe000-7f3abafff000 ---p 00000000 00:00 0 
7f3abafff000-7f3abb7ff000 rw-p 00000000 00:00 0 
7f3abb7ff000-7f3abb800000 ---p 00000000 00:00 0 
7f3abb800000-7f3abc000000 rw-p 00000000 00:00 0 
7f3abc000000-7f3abc15f000 rw-p 00000000 00:00 0 
7f3abc15f000-7f3ac0000000 ---p 00000000 00:00 0 
7f3ac0000000-7f3ac0024000 rw-p 00000000 00:00 0 
7f3ac0024000-7f3ac4000000 ---p 00000000 00:00 0 
7f3ac4000000-7f3ac41bd000 rw-p 00000000 00:00 0 
7f3ac41bd000-7f3ac8000000 ---p 00000000 00:00 0 
7f3ac8000000-7f3ac8159000 rw-p 00000000 00:00 0 
7f3ac8159000-7f3acc000000 ---p 00000000 00:00 0 
7f3acc000000-7f3acc15b000 rw-p 00000000 00:00 0 
7f3acc15b000-7f3ad0000000 ---p 00000000 00:00 0 
7f3ad0000000-7f3ad003d000 rw-p 00000000 00:00 0 
7f3ad003d000-7f3ad4000000 ---p 00000000 00:00 0 
7f3ad4000000-7f3ad4021000 rw-p 00000000 00:00 0 
7f3ad4021000-7f3ad8000000 ---p 00000000 00:00 0 
7f3ad8000000-7f3ad8159000 rw-p 00000000 00:00 0 
7f3ad8159000-7f3adc000000 ---p 00000000 00:00 0 
7f3adc7f9000-7f3adc7fa000 ---p 00000000 00:00 0 
7f3adc7fa000-7f3adcffa000 rw-p 00000000 00:00 0 
7f3adcffa000-7f3adcffb000 ---p 00000000 00:00 0 
7f3adcffb000-7f3add7fb000 rw-p 00000000 00:00 0 
7f3add7fb000-7f3add7fc000 ---p 00000000 00:00 0 
7f3add7fc000-7f3addffc000 rw-p 00000000 00:00 0 
7f3addffc000-7f3addffd000 ---p 00000000 00:00 0 
7f3addffd000-7f3ade7fd000 rw-p 00000000 00:00 0 
7f3ade7fd000-7f3ade7fe000 ---p 00000000 00:00 0 
7f3ade7fe000-7f3adeffe000 rw-p 00000000 00:00 0 
7f3adeffe000-7f3adefff000 ---p 00000000 00:00 0 
7f3adefff000-7f3adf7ff000 rw-p 00000000 00:00 0 
7f3adf7ff000-7f3adf800000 ---p 00000000 00:00 0 
7f3adf800000-7f3ae0000000 rw-p 00000000 00:00 0 
7f3ae0000000-7f3ae0050000 rw-p 00000000 00:00 0 
7f3ae0050000-7f3ae4000000 ---p 00000000 00:00 0 
7f3ae4000000-7f3ae4021000 rw-p 00000000 00:00 0 
7f3ae4021000-7f3ae8000000 ---p 00000000 00:00 0 
7f3ae8000000-7f3ae802a000 rw-p 00000000 00:00 0 
7f3ae802a000-7f3aec000000 ---p 00000000 00:00 0 
7f3aec000000-7f3aec021000 rw-p 00000000 00:00 0 
7f3aec021000-7f3af0000000 ---p 00000000 00:00 0 
7f3af07f9000-7f3af07fa000 ---p 00000000 00:00 0 
7f3af07fa000-7f3af0ffa000 rw-p 00000000 00:00 0 
7f3af0ffa000-7f3af0ffb000 ---p 00000000 00:00 0 
7f3af0ffb000-7f3af17fb000 rw-p 00000000 00:00 0 
7f3af17fb000-7f3af17fc000 ---p 00000000 00:00 0 
7f3af17fc000-7f3af1ffc000 rw-p 00000000 00:00 0 
7f3af1ffc000-7f3af1ffd000 ---p 00000000 00:00 0 
7f3af1ffd000-7f3af27fd000 rw-p 00000000 00:00 0 
7f3af27fd000-7f3af27fe000 ---p 00000000 00:00 0 
7f3af27fe000-7f3af2ffe000 rw-p 00000000 00:00 0 
7f3af2ffe000-7f3af2fff000 ---p 00000000 00:00 0 
7f3af2fff000-7f3af37ff000 rw-p 00000000 00:00 0 
7f3af37ff000-7f3af3800000 ---p 00000000 00:00 0 
7f3af3800000-7f3af4000000 rw-p 00000000 00:00 0 
7f3af4000000-7f3af4021000 rw-p 00000000 00:00 0 
7f3af4021000-7f3af8000000 ---p 00000000 00:00 0 
7f3af8000000-7f3af8476000 rw-p 00000000 00:00 0 
7f3af8476000-7f3afc000000 ---p 00000000 00:00 0 
7f3afc000000-7f3afc021000 rw-p 00000000 00:00 0 
7f3afc021000-7f3b00000000 ---p 00000000 00:00 0 
7f3b00000000-7f3b00021000 rw-p 00000000 00:00 0 
7f3b00021000-7f3b04000000 ---p 00000000 00:00 0 
7f3b047f9000-7f3b047fa000 ---p 00000000 00:00 0 
7f3b047fa000-7f3b04ffa000 rw-p 00000000 00:00 0 
7f3b04ffa000-7f3b04ffb000 ---p 00000000 00:00 0 
7f3b04ffb000-7f3b057fb000 rw-p 00000000 00:00 0 
7f3b057fb000-7f3b057fc000 ---p 00000000 00:00 0 
7f3b057fc000-7f3b05ffc000 rw-p 00000000 00:00 0 
7f3b05ffc000-7f3b05ffd000 ---p 00000000 00:00 0 
7f3b05ffd000-7f3b067fd000 rw-p 00000000 00:00 0 
7f3b067fd000-7f3b067fe000 ---p 00000000 00:00 0 
7f3b067fe000-7f3b06ffe000 rw-p 00000000 00:00 0 
7f3b06ffe000-7f3b06fff000 ---p 00000000 00:00 0 
7f3b06fff000-7f3b077ff000 rw-p 00000000 00:00 0 
7f3b077ff000-7f3b07800000 ---p 00000000 00:00 0 
7f3b07800000-7f3b08000000 rw-p 00000000 00:00 0 
7f3b08000000-7f3b08021000 rw-p 00000000 00:00 0 
7f3b08021000-7f3b0c000000 ---p 00000000 00:00 0 
7f3b0c000000-7f3b0c021000 rw-p 00000000 00:00 0 
7f3b0c021000-7f3b10000000 ---p 00000000 00:00 0 
7f3b10000000-7f3b10021000 rw-p 00000000 00:00 0 
7f3b10021000-7f3b14000000 ---p 00000000 00:00 0 
7f3b14000000-7f3b14021000 rw-p 00000000 00:00 0 
7f3b14021000-7f3b18000000 ---p 00000000 00:00 0 
7f3b18000000-7f3b18021000 rw-p 00000000 00:00 0 
7f3b18021000-7f3b1c000000 ---p 00000000 00:00 0 
7f3b1c000000-7f3b1c021000 rw-p 00000000 00:00 0 
7f3b1c021000-7f3b20000000 ---p 00000000 00:00 0 
7f3b207aa000-7f3b207ab000 ---p 00000000 00:00 0 
7f3b207ab000-7f3b20fab000 rw-p 00000000 00:00 0 
7f3b20fab000-7f3b20fac000 ---p 00000000 00:00 0 
7f3b20fac000-7f3b217ac000 rw-p 00000000 00:00 0 
7f3b217ac000-7f3b217ad000 ---p 00000000 00:00 0 
7f3b217ad000-7f3b21fad000 rw-p 00000000 00:00 0 
7f3b21fad000-7f3b21fae000 ---p 00000000 00:00 0 
7f3b21fae000-7f3b227ae000 rw-p 00000000 00:00 0 
7f3b227ae000-7f3b227af000 ---p 00000000 00:00 0 
7f3b227af000-7f3b22faf000 rw-p 00000000 00:00 0 
7f3b22faf000-7f3b22fb0000 ---p 00000000 00:00 0 
7f3b22fb0000-7f3b238b1000 rw-p 00000000 00:00 0 
7f3b238b1000-7f3b238b2000 ---p 00000000 00:00 0 
7f3b238b2000-7f3b240b2000 rw-p 00000000 00:00 0 
7f3b240b2000-7f3b240b3000 ---p 00000000 00:00 0 
7f3b240b3000-7f3b248b3000 rw-p 00000000 00:00 0 
7f3b248b3000-7f3b248b4000 ---p 00000000 00:00 0 
7f3b248b4000-7f3b250b4000 rw-p 00000000 00:00 0 
7f3b250b4000-7f3b250b6000 r-xp 00000000 103:02 7484213                   /usr/lib64/libfreebl3.so
7f3b250b6000-7f3b252b5000 ---p 00002000 103:02 7484213                   /usr/lib64/libfreebl3.so
7f3b252b5000-7f3b252b6000 r--p 00001000 103:02 7484213                   /usr/lib64/libfreebl3.so
7f3b252b6000-7f3b252b7000 rw-p 00002000 103:02 7484213                   /usr/lib64/libfreebl3.so
7f3b252b7000-7f3b25317000 r-xp 00000000 103:02 7484343                   /usr/lib64/libpcre.so.1.2.0
7f3b25317000-7f3b25517000 ---p 00060000 103:02 7484343                   /usr/lib64/libpcre.so.1.2.0
7f3b25517000-7f3b25518000 r--p 00060000 103:02 7484343                   /usr/lib64/libpcre.so.1.2.0
7f3b25518000-7f3b25519000 rw-p 00061000 103:02 7484343                   /usr/lib64/libpcre.so.1.2.0
7f3b25519000-7f3b25521000 r-xp 00000000 103:02 7484181                   /usr/lib64/libcrypt-2.17.so
7f3b25521000-7f3b25720000 ---p 00008000 103:02 7484181                   /usr/lib64/libcrypt-2.17.so
7f3b25720000-7f3b25721000 r--p 00007000 103:02 7484181                   /usr/lib64/libcrypt-2.17.so
7f3b25721000-7f3b25722000 rw-p 00008000 103:02 7484181                   /usr/lib64/libcrypt-2.17.so
7f3b25722000-7f3b25750000 rw-p 00000000 00:00 0 
7f3b25750000-7f3b25774000 r-xp 00000000 103:02 7484383                   /usr/lib64/libselinux.so.1
7f3b25774000-7f3b25973000 ---p 00024000 103:02 7484383                   /usr/lib64/libselinux.so.1
7f3b25973000-7f3b25974000 r--p 00023000 103:02 7484383                   /usr/lib64/libselinux.so.1
7f3b25974000-7f3b25975000 rw-p 00024000 103:02 7484383                   /usr/lib64/libselinux.so.1
7f3b25975000-7f3b25977000 rw-p 00000000 00:00 0 
7f3b25977000-7f3b25993000 r-xp 00000000 103:02 7484382                   /usr/lib64/libsasl2.so.3.0.0
7f3b25993000-7f3b25b92000 ---p 0001c000 103:02 7484382                   /usr/lib64/libsasl2.so.3.0.0
7f3b25b92000-7f3b25b93000 r--p 0001b000 103:02 7484382                   /usr/lib64/libsasl2.so.3.0.0
7f3b25b93000-7f3b25b94000 rw-p 0001c000 103:02 7484382                   /usr/lib64/libsasl2.so.3.0.0
7f3b25b94000-7f3b25baa000 r-xp 00000000 103:02 7484369                   /usr/lib64/libresolv-2.17.so
7f3b25baa000-7f3b25daa000 ---p 00016000 103:02 7484369                   /usr/lib64/libresolv-2.17.so
7f3b25daa000-7f3b25dab000 r--p 00016000 103:02 7484369                   /usr/lib64/libresolv-2.17.so
7f3b25dab000-7f3b25dac000 rw-p 00017000 103:02 7484369                   /usr/lib64/libresolv-2.17.so
7f3b25dac000-7f3b25dae000 rw-p 00000000 00:00 0 
7f3b25dae000-7f3b25db1000 r-xp 00000000 103:02 7484263                   /usr/lib64/libkeyutils.so.1.5
7f3b25db1000-7f3b25fb0000 ---p 00003000 103:02 7484263                   /usr/lib64/libkeyutils.so.1.5
7f3b25fb0000-7f3b25fb1000 r--p 00002000 103:02 7484263                   /usr/lib64/libkeyutils.so.1.5
7f3b25fb1000-7f3b25fb2000 rw-p 00003000 103:02 7484263                   /usr/lib64/libkeyutils.so.1.5
7f3b25fb2000-7f3b25fc0000 r-xp 00000000 103:02 8407470                   /usr/lib64/libkrb5support.so.0.1
7f3b25fc0000-7f3b261c0000 ---p 0000e000 103:02 8407470                   /usr/lib64/libkrb5support.so.0.1
7f3b261c0000-7f3b261c1000 r--p 0000e000 103:02 8407470                   /usr/lib64/libkrb5support.so.0.1
7f3b261c1000-7f3b261c2000 rw-p 0000f000 103:02 8407470                   /usr/lib64/libkrb5support.so.0.1
7f3b261c2000-7f3b26214000 r-xp 00000000 103:02 7484275                   /usr/lib64/libldap-2.4.so.2.10.7
7f3b26214000-7f3b26414000 ---p 00052000 103:02 7484275                   /usr/lib64/libldap-2.4.so.2.10.7
7f3b26414000-7f3b26416000 r--p 00052000 103:02 7484275                   /usr/lib64/libldap-2.4.so.2.10.7
7f3b26416000-7f3b26417000 rw-p 00054000 103:02 7484275                   /usr/lib64/libldap-2.4.so.2.10.7
7f3b26417000-7f3b26425000 r-xp 00000000 103:02 7484273                   /usr/lib64/liblber-2.4.so.2.10.7
7f3b26425000-7f3b26624000 ---p 0000e000 103:02 7484273                   /usr/lib64/liblber-2.4.so.2.10.7
7f3b26624000-7f3b26625000 r--p 0000d000 103:02 7484273                   /usr/lib64/liblber-2.4.so.2.10.7
7f3b26625000-7f3b26626000 rw-p 0000e000 103:02 7484273                   /usr/lib64/liblber-2.4.so.2.10.7
7f3b26626000-7f3b26660000 r-xp 00000000 103:02 7484304                   /usr/lib64/libnspr4.so
7f3b26660000-7f3b2685f000 ---p 0003a000 103:02 7484304                   /usr/lib64/libnspr4.so
7f3b2685f000-7f3b26860000 r--p 00039000 103:02 7484304                   /usr/lib64/libnspr4.so
7f3b26860000-7f3b26862000 rw-p 0003a000 103:02 7484304                   /usr/lib64/libnspr4.so
7f3b26862000-7f3b26864000 rw-p 00000000 00:00 0 
7f3b26864000-7f3b26868000 r-xp 00000000 103:02 7484352                   /usr/lib64/libplc4.so
7f3b26868000-7f3b26a67000 ---p 00004000 103:02 7484352                   /usr/lib64/libplc4.so
7f3b26a67000-7f3b26a68000 r--p 00003000 103:02 7484352                   /usr/lib64/libplc4.so
7f3b26a68000-7f3b26a69000 rw-p 00004000 103:02 7484352                   /usr/lib64/libplc4.so
7f3b26a69000-7f3b26a6c000 r-xp 00000000 103:02 7484353                   /usr/lib64/libplds4.so
7f3b26a6c000-7f3b26c6b000 ---p 00003000 103:02 7484353                   /usr/lib64/libplds4.so
7f3b26c6b000-7f3b26c6c000 r--p 00002000 103:02 7484353                   /usr/lib64/libplds4.so
7f3b26c6c000-7f3b26c6d000 rw-p 00003000 103:02 7484353                   /usr/lib64/libplds4.so
7f3b26c6d000-7f3b26c96000 r-xp 00000000 103:02 7484327                   /usr/lib64/libnssutil3.so
7f3b26c96000-7f3b26e95000 ---p 00029000 103:02 7484327                   /usr/lib64/libnssutil3.so
7f3b26e95000-7f3b26e9c000 r--p 00028000 103:02 7484327                   /usr/lib64/libnssutil3.so
7f3b26e9c000-7f3b26e9d000 rw-p 0002f000 103:02 7484327                   /usr/lib64/libnssutil3.so
7f3b26e9d000-7f3b26fc3000 r-xp 00000000 103:02 7484305                   /usr/lib64/libnss3.so
7f3b26fc3000-7f3b271c3000 ---p 00126000 103:02 7484305                   /usr/lib64/libnss3.so
7f3b271c3000-7f3b271c8000 r--p 00126000 103:02 7484305                   /usr/lib64/libnss3.so
7f3b271c8000-7f3b271ca000 rw-p 0012b000 103:02 7484305                   /usr/lib64/libnss3.so
7f3b271ca000-7f3b271cc000 rw-p 00000000 00:00 0 
7f3b271cc000-7f3b271f1000 r-xp 00000000 103:02 7484390                   /usr/lib64/libsmime3.so
7f3b271f1000-7f3b273f0000 ---p 00025000 103:02 7484390                   /usr/lib64/libsmime3.so
7f3b273f0000-7f3b273f3000 r--p 00024000 103:02 7484390                   /usr/lib64/libsmime3.so
7f3b273f3000-7f3b273f4000 rw-p 00027000 103:02 7484390                   /usr/lib64/libsmime3.so
7f3b273f4000-7f3b27447000 r-xp 00000000 103:02 7484399                   /usr/lib64/libssl3.so
7f3b27447000-7f3b27647000 ---p 00053000 103:02 7484399                   /usr/lib64/libssl3.so
7f3b27647000-7f3b2764b000 r--p 00053000 103:02 7484399                   /usr/lib64/libssl3.so
7f3b2764b000-7f3b2764c000 rw-p 00057000 103:02 7484399                   /usr/lib64/libssl3.so
7f3b2764c000-7f3b2764d000 rw-p 00000000 00:00 0 
7f3b2764d000-7f3b27678000 r-xp 00000000 103:02 7484396                   /usr/lib64/libssh2.so.1.0.1
7f3b27678000-7f3b27878000 ---p 0002b000 103:02 7484396                   /usr/lib64/libssh2.so.1.0.1
7f3b27878000-7f3b27879000 r--p 0002b000 103:02 7484396                   /usr/lib64/libssh2.so.1.0.1
7f3b27879000-7f3b2787a000 rw-p 0002c000 103:02 7484396                   /usr/lib64/libssh2.so.1.0.1
7f3b2787a000-7f3b278ac000 r-xp 00000000 103:02 7484253                   /usr/lib64/libidn.so.11.6.11
7f3b278ac000-7f3b27aab000 ---p 00032000 103:02 7484253                   /usr/lib64/libidn.so.11.6.11
7f3b27aab000-7f3b27aac000 r--p 00031000 103:02 7484253                   /usr/lib64/libidn.so.11.6.11
7f3b27aac000-7f3b27aad000 rw-p 00032000 103:02 7484253                   /usr/lib64/libidn.so.11.6.11
7f3b27aad000-7f3b27ade000 r-xp 00000000 103:02 8407449                   /usr/lib64/libk5crypto.so.3.1
7f3b27ade000-7f3b27cdd000 ---p 00031000 103:02 8407449                   /usr/lib64/libk5crypto.so.3.1
7f3b27cdd000-7f3b27cdf000 r--p 00030000 103:02 8407449                   /usr/lib64/libk5crypto.so.3.1
7f3b27cdf000-7f3b27ce0000 rw-p 00032000 103:02 8407449                   /usr/lib64/libk5crypto.so.3.1
7f3b27ce0000-7f3b27ce3000 r-xp 00000000 103:02 8407435                   /usr/lib64/libcom_err.so.2.1
7f3b27ce3000-7f3b27ee2000 ---p 00003000 103:02 8407435                   /usr/lib64/libcom_err.so.2.1
7f3b27ee2000-7f3b27ee3000 r--p 00002000 103:02 8407435                   /usr/lib64/libcom_err.so.2.1
7f3b27ee3000-7f3b27ee4000 rw-p 00003000 103:02 8407435                   /usr/lib64/libcom_err.so.2.1
7f3b27ee4000-7f3b27fbd000 r-xp 00000000 103:02 8407467                   /usr/lib64/libkrb5.so.3.3
7f3b27fbd000-7f3b281bc000 ---p 000d9000 103:02 8407467                   /usr/lib64/libkrb5.so.3.3
7f3b281bc000-7f3b281ca000 r--p 000d8000 103:02 8407467                   /usr/lib64/libkrb5.so.3.3
7f3b281ca000-7f3b281cd000 rw-p 000e6000 103:02 8407467                   /usr/lib64/libkrb5.so.3.3
7f3b281cd000-7f3b28217000 r-xp 00000000 103:02 8407443                   /usr/lib64/libgssapi_krb5.so.2.2
7f3b28217000-7f3b28417000 ---p 0004a000 103:02 8407443                   /usr/lib64/libgssapi_krb5.so.2.2
7f3b28417000-7f3b28418000 r--p 0004a000 103:02 8407443                   /usr/lib64/libgssapi_krb5.so.2.2
7f3b28418000-7f3b2841a000 rw-p 0004b000 103:02 8407443                   /usr/lib64/libgssapi_krb5.so.2.2
7f3b2841a000-7f3b286f8000 r-xp 00000000 103:02 8407486                   /usr/lib64/mysql/libmysqlclient.so.18.0.0
7f3b286f8000-7f3b288f7000 ---p 002de000 103:02 8407486                   /usr/lib64/mysql/libmysqlclient.so.18.0.0
7f3b288f7000-7f3b28900000 r--p 002dd000 103:02 8407486                   /usr/lib64/mysql/libmysqlclient.so.18.0.0
7f3b28900000-7f3b28914000 rw-p 002e6000 103:02 8407486                   /usr/lib64/mysql/libmysqlclient.so.18.0.0
7f3b28914000-7f3b2891a000 rw-p 00000000 00:00 0 
7f3b2891a000-7f3b28955000 r-xp 00000000 103:02 8407477                   /usr/lib64/libquadmath.so.0.0.0
7f3b28955000-7f3b28b54000 ---p 0003b000 103:02 8407477                   /usr/lib64/libquadmath.so.0.0.0
7f3b28b54000-7f3b28b55000 r--p 0003a000 103:02 8407477                   /usr/lib64/libquadmath.so.0.0.0
7f3b28b55000-7f3b28b56000 rw-p 0003b000 103:02 8407477                   /usr/lib64/libquadmath.so.0.0.0
7f3b28b56000-7f3b28d19000 r-xp 00000000 103:02 7484169                   /usr/lib64/libc-2.17.so
7f3b28d19000-7f3b28f19000 ---p 001c3000 103:02 7484169                   /usr/lib64/libc-2.17.so
7f3b28f19000-7f3b28f1d000 r--p 001c3000 103:02 7484169                   /usr/lib64/libc-2.17.so
7f3b28f1d000-7f3b28f1f000 rw-p 001c7000 103:02 7484169                   /usr/lib64/libc-2.17.so
7f3b28f1f000-7f3b28f24000 rw-p 00000000 00:00 0 
7f3b28f24000-7f3b28f39000 r-xp 00000000 103:02 7484216                   /usr/lib64/libgcc_s-4.8.5-20150702.so.1
7f3b28f39000-7f3b29138000 ---p 00015000 103:02 7484216                   /usr/lib64/libgcc_s-4.8.5-20150702.so.1
7f3b29138000-7f3b29139000 r--p 00014000 103:02 7484216                   /usr/lib64/libgcc_s-4.8.5-20150702.so.1
7f3b29139000-7f3b2913a000 rw-p 00015000 103:02 7484216                   /usr/lib64/libgcc_s-4.8.5-20150702.so.1
7f3b2913a000-7f3b2923b000 r-xp 00000000 103:02 7484283                   /usr/lib64/libm-2.17.so
7f3b2923b000-7f3b2943a000 ---p 00101000 103:02 7484283                   /usr/lib64/libm-2.17.so
7f3b2943a000-7f3b2943b000 r--p 00100000 103:02 7484283                   /usr/lib64/libm-2.17.so
7f3b2943b000-7f3b2943c000 rw-p 00101000 103:02 7484283                   /usr/lib64/libm-2.17.so
7f3b2943c000-7f3b29525000 r-xp 00000000 103:02 7484401                   /usr/lib64/libstdc++.so.6.0.19
7f3b29525000-7f3b29724000 ---p 000e9000 103:02 7484401                   /usr/lib64/libstdc++.so.6.0.19
7f3b29724000-7f3b2972c000 r--p 000e8000 103:02 7484401                   /usr/lib64/libstdc++.so.6.0.19
7f3b2972c000-7f3b2972e000 rw-p 000f0000 103:02 7484401                   /usr/lib64/libstdc++.so.6.0.19
7f3b2972e000-7f3b29743000 rw-p 00000000 00:00 0 
7f3b29743000-7f3b29749000 r-xp 00000000 103:02 8927671                   /var/lib/milvus/lib/libfiu.so.1.00
7f3b29749000-7f3b29948000 ---p 00006000 103:02 8927671                   /var/lib/milvus/lib/libfiu.so.1.00
7f3b29948000-7f3b29949000 r--p 00005000 103:02 8927671                   /var/lib/milvus/lib/libfiu.so.1.00
7f3b29949000-7f3b2994a000 rw-p 00006000 103:02 8927671                   /var/lib/milvus/lib/libfiu.so.1.00
7f3b2994a000-7f3b299b0000 r-xp 00000000 103:02 7484190                   /usr/lib64/libcurl.so.4.3.0
7f3b299b0000-7f3b29bb0000 ---p 00066000 103:02 7484190                   /usr/lib64/libcurl.so.4.3.0
7f3b29bb0000-7f3b29bb2000 r--p 00066000 103:02 7484190                   /usr/lib64/libcurl.so.4.3.0
7f3b29bb2000-7f3b29bb3000 rw-p 00068000 103:02 7484190                   /usr/lib64/libcurl.so.4.3.0
7f3b29bb3000-7f3b29bb4000 rw-p 00000000 00:00 0 
7f3b29bb4000-7f3b29bcb000 r-xp 00000000 103:02 7484360                   /usr/lib64/libpthread-2.17.so
7f3b29bcb000-7f3b29dca000 ---p 00017000 103:02 7484360                   /usr/lib64/libpthread-2.17.so
7f3b29dca000-7f3b29dcb000 r--p 00016000 103:02 7484360                   /usr/lib64/libpthread-2.17.so
7f3b29dcb000-7f3b29dcc000 rw-p 00017000 103:02 7484360                   /usr/lib64/libpthread-2.17.so
7f3b29dcc000-7f3b29dd0000 rw-p 00000000 00:00 0 
7f3b29dd0000-7f3b29dd7000 r-xp 00000000 103:02 7484379                   /usr/lib64/librt-2.17.so
7f3b29dd7000-7f3b29fd6000 ---p 00007000 103:02 7484379                   /usr/lib64/librt-2.17.so
7f3b29fd6000-7f3b29fd7000 r--p 00006000 103:02 7484379                   /usr/lib64/librt-2.17.so
7f3b29fd7000-7f3b29fd8000 rw-p 00007000 103:02 7484379                   /usr/lib64/librt-2.17.so
7f3b29fd8000-7f3b29fda000 r-xp 00000000 103:02 7484198                   /usr/lib64/libdl-2.17.so
7f3b29fda000-7f3b2a1da000 ---p 00002000 103:02 7484198                   /usr/lib64/libdl-2.17.so
7f3b2a1da000-7f3b2a1db000 r--p 00002000 103:02 7484198                   /usr/lib64/libdl-2.17.so
7f3b2a1db000-7f3b2a1dc000 rw-p 00003000 103:02 7484198                   /usr/lib64/libdl-2.17.so
7f3b2a1dc000-7f3b2a412000 r-xp 00000000 103:02 7484183                   /usr/lib64/libcrypto.so.1.0.2k
7f3b2a412000-7f3b2a612000 ---p 00236000 103:02 7484183                   /usr/lib64/libcrypto.so.1.0.2k
7f3b2a612000-7f3b2a62e000 r--p 00236000 103:02 7484183                   /usr/lib64/libcrypto.so.1.0.2k
7f3b2a62e000-7f3b2a63b000 rw-p 00252000 103:02 7484183                   /usr/lib64/libcrypto.so.1.0.2k
7f3b2a63b000-7f3b2a63f000 rw-p 00000000 00:00 0 
7f3b2a63f000-7f3b2a6a6000 r-xp 00000000 103:02 7484397                   /usr/lib64/libssl.so.1.0.2k
7f3b2a6a6000-7f3b2a8a6000 ---p 00067000 103:02 7484397                   /usr/lib64/libssl.so.1.0.2k
7f3b2a8a6000-7f3b2a8aa000 r--p 00067000 103:02 7484397                   /usr/lib64/libssl.so.1.0.2k
7f3b2a8aa000-7f3b2a8b1000 rw-p 0006b000 103:02 7484397                   /usr/lib64/libssl.so.1.0.2k
7f3b2a8b1000-7f3b2a8c8000 r-xp 00000000 103:02 7484302                   /usr/lib64/libnsl-2.17.so
7f3b2a8c8000-7f3b2aac7000 ---p 00017000 103:02 7484302                   /usr/lib64/libnsl-2.17.so
7f3b2aac7000-7f3b2aac8000 r--p 00016000 103:02 7484302                   /usr/lib64/libnsl-2.17.so
7f3b2aac8000-7f3b2aac9000 rw-p 00017000 103:02 7484302                   /usr/lib64/libnsl-2.17.so
7f3b2aac9000-7f3b2aacb000 rw-p 00000000 00:00 0 
7f3b2aacb000-7f3b2ab35000 r-xp 00000000 103:02 8927674                   /var/lib/milvus/lib/libmysqlpp.so.3.2.4
7f3b2ab35000-7f3b2ad35000 ---p 0006a000 103:02 8927674                   /var/lib/milvus/lib/libmysqlpp.so.3.2.4
7f3b2ad35000-7f3b2ad37000 r--p 0006a000 103:02 8927674                   /var/lib/milvus/lib/libmysqlpp.so.3.2.4
7f3b2ad37000-7f3b2ad38000 rw-p 0006c000 103:02 8927674                   /var/lib/milvus/lib/libmysqlpp.so.3.2.4
7f3b2ad38000-7f3b2ad39000 rw-p 00000000 00:00 0 
7f3b2ad39000-7f3b2ad4e000 r-xp 00000000 103:02 7484437                   /usr/lib64/libz.so.1.2.7
7f3b2ad4e000-7f3b2af4d000 ---p 00015000 103:02 7484437                   /usr/lib64/libz.so.1.2.7
7f3b2af4d000-7f3b2af4e000 r--p 00014000 103:02 7484437                   /usr/lib64/libz.so.1.2.7
7f3b2af4e000-7f3b2af4f000 rw-p 00015000 103:02 7484437                   /usr/lib64/libz.so.1.2.7
7f3b2af4f000-7f3b2af74000 r-xp 00000000 103:02 8407440                   /usr/lib64/libgomp.so.1.0.0
7f3b2af74000-7f3b2b173000 ---p 00025000 103:02 8407440                   /usr/lib64/libgomp.so.1.0.0
7f3b2b173000-7f3b2b174000 r--p 00024000 103:02 8407440                   /usr/lib64/libgomp.so.1.0.0
7f3b2b174000-7f3b2b175000 rw-p 00025000 103:02 8407440                   /usr/lib64/libgomp.so.1.0.0
7f3b2b175000-7f3b2b24b000 r-xp 00000000 103:02 8927678                   /var/lib/milvus/lib/libngt.so.1.12.0
7f3b2b24b000-7f3b2b44b000 ---p 000d6000 103:02 8927678                   /var/lib/milvus/lib/libngt.so.1.12.0
7f3b2b44b000-7f3b2b44d000 r--p 000d6000 103:02 8927678                   /var/lib/milvus/lib/libngt.so.1.12.0
7f3b2b44d000-7f3b2b44f000 rw-p 000d8000 103:02 8927678                   /var/lib/milvus/lib/libngt.so.1.12.0
7f3b2b44f000-7f3b2b629000 r-xp 00000000 103:02 8407438                   /usr/lib64/libgfortran.so.4.0.0
7f3b2b629000-7f3b2b828000 ---p 001da000 103:02 8407438                   /usr/lib64/libgfortran.so.4.0.0
7f3b2b828000-7f3b2b829000 r--p 001d9000 103:02 8407438                   /usr/lib64/libgfortran.so.4.0.0
7f3b2b829000-7f3b2b82a000 rw-p 001da000 103:02 8407438                   /usr/lib64/libgfortran.so.4.0.0
7f3b2b82a000-7f3b2b82b000 rw-p 00000000 00:00 0 
7f3b2b82b000-7f3b2debe000 r-xp 00000000 103:02 8927704                   /var/lib/milvus/lib/libopenblas-r0.3.9.so
7f3b2debe000-7f3b2e0be000 ---p 02693000 103:02 8927704                   /var/lib/milvus/lib/libopenblas-r0.3.9.so
7f3b2e0be000-7f3b2e0c2000 r--p 02693000 103:02 8927704                   /var/lib/milvus/lib/libopenblas-r0.3.9.so
7f3b2e0c2000-7f3b2e0dd000 rw-p 02697000 103:02 8927704                   /var/lib/milvus/lib/libopenblas-r0.3.9.so
7f3b2e0dd000-7f3b2e0e3000 rw-p 00000000 00:00 0 
7f3b2e0e3000-7f3b2e105000 r-xp 00000000 103:02 7484145                   /usr/lib64/ld-2.17.so
7f3b2e1dd000-7f3b2e2de000 rw-p 00000000 00:00 0 
7f3b2e2de000-7f3b2e2e6000 rw-s 00000000 103:02 9054412                   /var/lib/milvus/db/meta.sqlite-shm
7f3b2e2e6000-7f3b2e2ff000 rw-p 00000000 00:00 0 
7f3b2e300000-7f3b2e304000 rw-p 00000000 00:00 0 
7f3b2e304000-7f3b2e305000 r--p 00021000 103:02 7484145                   /usr/lib64/ld-2.17.so
7f3b2e305000-7f3b2e306000 rw-p 00022000 103:02 7484145                   /usr/lib64/ld-2.17.so
7f3b2e306000-7f3b2e307000 rw-p 00000000 00:00 0 
7ffd565d8000-7ffd565f9000 rw-p 00000000 00:00 0                          [stack]
7ffd565fb000-7ffd565fe000 r--p 00000000 00:00 0                          [vvar]
7ffd565fe000-7ffd565ff000 r-xp 00000000 00:00 0                          [vdso]
ffffffffff600000-ffffffffff601000 --xp 00000000 00:00 0                  [vsyscall]
No.0: /var/lib/milvus/bin/milvus_server(_ZN6milvus15PrintStacktraceEv+0x26) [0x111e866]
No.1: /var/lib/milvus/bin/milvus_server(_ZN6milvus12HandleSignalEi+0x190) [0x111f3d0]
No.2: /lib64/libc.so.6(+0x36400) [0x7f3b28b8c400]
No.3: /lib64/libc.so.6(abort+0x297) [0x7f3b28b8dbc7]
No.4: /lib64/libc.so.6(+0x78ed7) [0x7f3b28bceed7]
No.5: /lib64/libc.so.6(+0x7f3e4) [0x7f3b28bd53e4]
No.6: /lib64/libc.so.6(+0x82b20) [0x7f3b28bd8b20]
No.7: /lib64/libc.so.6(__libc_malloc+0x4c) [0x7f3b28bdb6fc]
No.8: /lib64/libstdc++.so.6(_Znwm+0x1d) [0x7f3b2949aecd]
No.9: /var/lib/milvus/bin/milvus_server(_ZN6milvus6engine8snapshot16OperationContextC2ERKS2_+0x4c2) [0x93bd62]
No.10: /var/lib/milvus/bin/milvus_server(_ZN6milvus6engine8snapshot10OperationsC2ERKNS1_16OperationContextENS1_14ScopedResourceINS1_8SnapshotEEERKNS1_14OperationsTypeE+0x41) [0x996421]
No.11: /var/lib/milvus/bin/milvus_server(_ZN6milvus6engine8snapshot14MergeOperationC2ERKNS1_16OperationContextENS1_14ScopedResourceINS1_8SnapshotEEE+0x4b) [0x95166b]
No.12: /var/lib/milvus/bin/milvus_server(_ZN6milvus6engine9MergeTask7ExecuteEv+0x54c) [0x94b29c]
No.13: /var/lib/milvus/bin/milvus_server(_ZN6milvus6engine6DBImpl7CompactERKSt10shared_ptrINS_6server7ContextEERKSsd+0x8da) [0x8f2aba]
No.14: /var/lib/milvus/bin/milvus_server(_ZN6milvus6engine7DBProxy7CompactERKSt10shared_ptrINS_6server7ContextEERKSsd+0xb3) [0xa60a03]
No.15: /var/lib/milvus/bin/milvus_server(_ZN6milvus6server10CompactReq9OnExecuteEv+0x30c) [0x8cd76c]
No.16: /var/lib/milvus/bin/milvus_server(_ZN6milvus6server7BaseReq7ExecuteEv+0x20) [0x8cb010]
No.17: /var/lib/milvus/bin/milvus_server(_ZN6milvus6server12ReqScheduler13TakeToExecuteESt10shared_ptrINS0_8ReqQueueEE+0x12d) [0x8c75cd]
No.18: /var/lib/milvus/bin/milvus_server(_ZNSt6thread11_State_implINS_8_InvokerISt5tupleIJMN6milvus6server12ReqSchedulerEFvSt10shared_ptrINS4_8ReqQueueEEEPS5_S8_EEEEE6_M_runEv+0x47) [0x8c9b77]
No.19: /var/lib/milvus/bin/milvus_server() [0x112c7ef]
No.20: /lib64/libpthread.so.0(+0x7ea5) [0x7f3b29bbbea5]
No.21: /lib64/libc.so.6(clone+0x6d) [0x7f3b28c548dd]
Milvus server is going to shutdown ...

```

**Expected behavior**
A clear and concise description of what you expected to happen.

**Method of installation**
- [ ] Docker/cpu
- [ ] Docker/gpu
- [ ] Build from source

**Environment details**
- Hardware/Softwars conditions (OS, CPU, GPU, Memory)


- Milvus version (master or released version)


**Configuration file**
Settings you made in `server_config.yaml` or `milvus.yaml`

```yaml
paste-file-content-here
```

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Additional context**
Add any other context about the problem here.

--- a/core/src/db/snapshot/Snapshot.h
+++ b/core/src/db/snapshot/Snapshot.h
@@ -273,7 +273,6 @@ class Snapshot : public ReferenceProxy {
         ReferenceProxy::UnRef();
         if (ref_count_ == 0) {
             UnRefAll();
-            std::apply([](auto&... holder) { ((holder.clear()), ...); }, resources_);
         }
     }
 

--- a/core/src/segment/SegmentReader.cpp
+++ b/core/src/segment/SegmentReader.cpp
@@ -671,7 +671,7 @@ SegmentReader::LoadDeletedDocs(segment::DeletedDocsPtr& deleted_docs_ptr) {
             auto id = segment_visitor_->GetSegment()->GetID();
             auto sc = segment_visitor_->GetSnapshot()->GetSegmentCommitBySegmentId(id);
             if (sc != nullptr && deleted_docs_ptr != nullptr) {
-                deleted_docs_ptr->GenBlacklist(sc->GetRowCount() + deleted_docs_ptr->Size());
+                deleted_docs_ptr->GenBlacklist(sc->GetRowCount() + deleted_docs_ptr->GetCount());
             }
             cache::CpuCacheMgr::GetInstance().InsertItem(file_path, deleted_docs_ptr);  // put into cache
         } else {

--- a/core/src/segment/SegmentReader.cpp
+++ b/core/src/segment/SegmentReader.cpp
@@ -670,6 +670,8 @@ SegmentReader::LoadDeletedDocs(segment::DeletedDocsPtr& deleted_docs_ptr) {
             STATUS_CHECK(ss_codec.GetDeletedDocsFormat()->Read(fs_ptr_, file_path, deleted_docs_ptr));
             auto id = segment_visitor_->GetSegment()->GetID();
             auto sc = segment_visitor_->GetSnapshot()->GetSegmentCommitBySegmentId(id);
+            // The black list size must be equal to total entity count containing deleted count
+            // and segment row count.
             if (sc != nullptr && deleted_docs_ptr != nullptr) {
                 deleted_docs_ptr->GenBlacklist(sc->GetRowCount() + deleted_docs_ptr->GetCount());
             }

--- a/core/src/segment/SegmentReader.cpp
+++ b/core/src/segment/SegmentReader.cpp
@@ -670,8 +670,8 @@ SegmentReader::LoadDeletedDocs(segment::DeletedDocsPtr& deleted_docs_ptr) {
             STATUS_CHECK(ss_codec.GetDeletedDocsFormat()->Read(fs_ptr_, file_path, deleted_docs_ptr));
             auto id = segment_visitor_->GetSegment()->GetID();
             auto sc = segment_visitor_->GetSnapshot()->GetSegmentCommitBySegmentId(id);
-            if (sc) {
-                deleted_docs_ptr->GenBlacklist(sc->GetRowCount());
+            if (sc != nullptr && deleted_docs_ptr != nullptr) {
+                deleted_docs_ptr->GenBlacklist(sc->GetRowCount() + deleted_docs_ptr->Size());
             }
             cache::CpuCacheMgr::GetInstance().InsertItem(file_path, deleted_docs_ptr);  // put into cache
         } else {

